code/obj/item/kitchen.dm

```diff
@@ -758,6 +773,8 @@ TRAYS
 
 		src.visible_message("\The [src] looks less sturdy now.")
 
+	unique_tap_garbage_fluck(mob/M as mob, mob/user as mob)
```

Comment:
Unintentional?

Reply:
it was intentional, cause its a bad proc name i made an even badder one :)

Reply:
Mbc why 😿